Tests/MMEAPIConfigTests.m

```diff
@@ -49,6 +49,8 @@ - (void) test001_TestHostConfigURL {
 }
 
 - (void) test002_StaringConfigUdpate {
+    XCTSkipIf(([[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){.majorVersion = 14, .minorVersion = 0, .patchVersion = 0}]), @"Skip, since it is not working on iOS14");
```

Comment:
As a reminder to ourselves, can you add in the description what `it` means? `CFSocketInvalidate` crashes with `EXC_GUARD` on iOS 14

Reply:
> As a reminder to ourselves, can you add in the description what `it` means? `CFSocketInvalidate` crashes with `EXC_GUARD` on iOS 14

Good point.